File: testdata/changeSignature/fromScala/AddNewClause_after.scala
class AddNewClause {
  def foo(b: Boolean)(x: Int, y: Int) = {}

  foo(true)(1, 0)
  this.foo(true)(1, 0)
}

class AddNewClause2 extends AddNewClause {
  override def foo(b: Boolean)(x: Int, y: Int): Unit = {
    super.foo(true)(x, 0)
  }
}

File: scala/scala-impl/testdata/changeSignature/fromScala/AddNewClause_after.java
public class AddNewClauseJava extends AddNewClause {
    @Override
    public void foo(boolean b, int x, int y) {
        AddNewClause a = new AddNewClause();
        a.foo(true, 1, 0);


        super.foo(true, x, 0);
    }
}
